android/app/build.gradle

```diff
@@ -53,12 +53,12 @@ android {
     }
 
     signingConfigs {
-        debug  {
-            storeFile file('keystore.jks')
-            storePassword 'tech.pylons'
-            keyAlias 'key_tech_pylon'
-            keyPassword 'tech.pylons'
-        }
+//        debug  {
```

Comment:
Is the debug code necessary, or is it dead code?

Reply:
We un comment this when we need to release the build.  This makes the build a little bit faster during testing.